geowebcache/wmts/src/test/java/org/geowebcache/service/wmts/WMTSServiceTest.java

```diff
@@ -840,9 +839,11 @@ public void testGetCapEmptyStyleFilter() throws Exception {
 
         String result = resp.getContentAsString();
 
-        Validator validator = new Validator(result);
-        validator.useXMLSchema(true);
-        validator.assertIsValid();
+        System.setProperty("http.agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36");
```

Comment:
is it needed?
what about the validators commented out section ?
(wondering if that was part of a debug session)

Reply:
Good catch, I was trying to have the validator use a different user agent, as xlink.xsd was available from a browser.